addons/mail/models/mail_thread.py

```diff
@@ -196,6 +196,8 @@ def _get_message_unread(self):
     @api.multi
     def _get_message_needaction(self):
         res = dict((res_id, 0) for res_id in self.ids)
```

Comment:
what happens if you have done instead the following fix?
```python
res = dict((res_id, 0) for res_id in self.ids or self._ids)
```

Reply:
In the very specific use-case where the fix appeared to be necessary, the call on the id is done anyway, so I assumed it was best to go this way.

Is there a use-case where this would be useful?